main.js

```diff
@@ -32,3 +33,218 @@ const toggleMenu = () => {
+    const companyType = document.createElement('span');
+    companyType.classList.add('att2');
+    companyType.innerText = item.company_type;
+    workatt.appendChild(companyType);
+    const secDotImg = document.createElement('img');
+    secDotImg.classList.add('dot');
+    secDotImg.src = './images/Dot.png';
+    secDotImg.alt = 'dot';
+    workatt.appendChild(secDotImg);
+    const companyDate = document.createElement('span');
+    companyDate.classList.add('att2');
+    companyDate.innerText = item.company_year;
+    workatt.appendChild(companyDate);
+    workbody.appendChild(workatt);
+
+    const workDescription = document.createElement('p');
+    workDescription.classList.add('card-text');
+    workDescription.innerText = item.description;
+    workbody.appendChild(workDescription);
+    const projectTags = document.createElement('ul');
+    projectTags.classList.add('tag-section');
+    item.tags.forEach((tech) => {
+      const techItem = document.createElement('li');
+      techItem.classList.add('tag-div');
+      const tag = document.createElement('span');
+      tag.classList.add('tag');
+      tag.innerText = tech;
+      techItem.appendChild(tag);
+      projectTags.appendChild(techItem);
+    });
+
+    workbody.appendChild(projectTags);
+    const seeMoreBtn = document.createElement('button');
+    seeMoreBtn.type = 'button';
+    seeMoreBtn.classList.add('button');
+    seeMoreBtn.innerText = 'See Project';
+    seeMoreBtn.dataset.key = item.key;
+    seeMoreBtn.id = `myBtn${item.key}`;
+    workbody.appendChild(seeMoreBtn);
+    list.appendChild(workbody);
+    CardWrap.appendChild(list);
+  });
+};
+
+lunchCards();
+
+const modalTitle = document.querySelector('#myModal h2');
+const modalImg = document.querySelector('#myModal #pic1');
+const modalDesc = document.querySelector('#myModal .description');
+const modaltag1 = document.querySelector('#myModal .tag1');
+const modaltag2 = document.querySelector('#myModal .tag2');
+const modaltag3 = document.querySelector('#myModal .tag3');
+const modaltag4 = document.querySelector('#myModal .tag4');
+const modallive = document.querySelector('#myModal #live1');
+const modalgithub = document.querySelector('#myModal #github1');
+
+const modal = document.getElementById('myModal');
+const btn1 = document.getElementById('myBtn1');
+const btn2 = document.getElementById('myBtn2');
+const btn3 = document.getElementById('myBtn3');
+const btn4 = document.getElementById('myBtn4');
+
+const span = document.getElementsByClassName('close')[0];
+
+btn1.onclick = function () {
+  modal.style.display = 'block';
+  modalTitle.textContent = cards[0].title;
+  modalDesc.textContent = cards[0].description;
+  modalImg.src = cards[0].img;
+  modaltag1.textContent = 'HTML';
+  modaltag2.textContent = 'Ruby on Rails';
+  modaltag3.textContent = 'CSS';
+  modaltag4.textContent = 'JavaScript';
+  modallive.href = cards[0].liveDemoLink;
+  modalgithub.href = cards[0].githubLink;
+};
```

Comment:
- Please for the mobile view make the popup container `fixed so that when a user scrolls the page the popup won't go out of view.
- Please make the text description of the project have a font weight of `400` as it is in the Figma design.

##### Your design
![Screenshot from 2022-06-15 11-58-04](https://user-images.githubusercontent.com/58771507/173812103-d6f0870d-517c-41fd-bf1d-25244487d434.png)

##### Figma design 
![Screenshot from 2022-06-15 12-02-10](https://user-images.githubusercontent.com/58771507/173812243-b8e3836b-74fd-4fee-96e3-490929fc6dbb.png)